azurerm/internal/services/securitycenter/resource_arm_security_center_setting.go

```diff
@@ -0,0 +1,112 @@
+package securitycenter
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/preview/security/mgmt/v3.0/security"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/validation"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/clients"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/securitycenter/azuresdkhacks"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/services/securitycenter/parse"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/internal/timeouts"
+)
+
+func resourceArmSecurityCenterSetting() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmSecurityCenterSettingUpdate,
+		Read:   resourceArmSecurityCenterSettingRead,
+		Update: resourceArmSecurityCenterSettingUpdate,
+		Delete: resourceArmSecurityCenterSettingDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(60 * time.Minute),
+			Read:   schema.DefaultTimeout(5 * time.Minute),
+			Update: schema.DefaultTimeout(60 * time.Minute),
+			Delete: schema.DefaultTimeout(60 * time.Minute),
```

Comment:
Could these timeouts be shorter?

Reply:
Good question, how do you assess the value that the timeouts should have? In general, creating a resource doesn't take 30-60 mins, however, most of the resources seem to have this timeout.

Reply:
I don't believe there is a set formula. We err on the side of caution, you might surprised how long operations can sometimes take :D

Given this is (I think?) a straightforward logical setting, keeping it down to 5-10 mins should be safe?